apps/richie/templates/nginx/route.yml.j2

```diff
@@ -0,0 +1,43 @@
+{%- set host = richie_host -%}
+
+{#
+    Host prefix can be: previous, next or current. You can use this template with
+    either a prefix set using the following variables (in precedence order):
+
+        - item.1 (in a nested loop)
```

Comment:
I don't understand "in a nested loop"...

Reply:
can item be a namedtuple of a dictionary so we write something else than "item.1" !? Can item have a better name also ? it's an item of what?

Reply:
I'm referring to Ansible nested loops: https://docs.ansible.com/ansible/2.5/user_guide/playbooks_loops.html#complex-loops

But I got your point and I'll come with a better idea / implementation.